src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -174,13 +174,8 @@ void BaseQNAcceleration::initialize(
     }
   }
 
-  // Append old value columns, if not done outside of acceleration already
-  for (DataMap::value_type &pair : cplData) {
-    int cols = pair.second->oldValues.cols();
-    if (cols < 1) { // Add only, if not already done
-      //PRECICE_ASSERT(pair.second->values().size() > 0, pair.first);
-      utils::append(pair.second->oldValues, (Eigen::VectorXd) Eigen::VectorXd::Zero(pair.second->values().size()));
-    }
+  for (DataMap::value_type &pair : cplData) { // @todo: seems to be unnecessary, since this is again just a loop over CouplingScheme::getAccelerationData
```

Comment:
see above

Reply:
Removing this part was tricky. See 7ab0220a248cd87138f6f8235d3e03f006101083. I think it's good to not initialize the CouplingData in the Acceleration. This should be done before. I will close this discussion, since it's resolved by 7ab0220a248cd87138f6f8235d3e03f006101083, but will comment on the new problem I see in `BaseCouplingScheme::setupDataMatrices` somewhere else.